CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 4.18.0 (Unreleased)
+
+ENHANCEMENTS:
+
+* dependencies: `appconfiguration` - update to API version `2024-05-01` [GH-28700]
+* dependencies: update `azurerm_cdn_frontdoor_rule` to API version `2024-02-01` [GH-28308]
+* dependencies: update `azurerm_cdn_frontdoor_ruleset` to API version `2024-02-01` [GH-28308]
+* dependencies: update `go-azure-sdk` to `v0.20250131.1134653` [GH-28674]
+* Data Source: `azurerm_cdn_frontdoor_firewall_policy` - add support for `js_challenge_cookie_expiration_in_minutes` policy [GH-28284]
+* Data Source: `azurerm_nginx_configuration` - add support for the `protected_file.content_hash` property [GH-28532]
+* `azurerm_cdn_frontdoor_firewall_policy` - add support for `js_challenge_cookie_expiration_in_minutes` policy [GH-28284]
```

Comment:
I think we also need to add a bullet point for the `JSChallenge` `action` type for the `managed_rule` `override` block?
```suggestion
* `azurerm_cdn_frontdoor_firewall_policy` - add support for `js_challenge_cookie_expiration_in_minutes` policy [GH-28284]
* `azurerm_cdn_frontdoor_firewall_policy` - add support for `JSChallenge` action in the `managed_rule` `override` block [GH-28308]
```